File: Tema 3 -Sentencia condicional (if-switch)/ejercicios/ejercicio1 dia de la semana y que materia toca/archives/1.py
import sys

# Obtener los argumentos de la línea de comandos
num1 = (sys.argv[1])
print(type(num1))
if num1 == "lunes":
    mensaje = "Lunes tienes matematicas"
elif num1 == "martes":
    mensaje = "Martes tienes lenguaje"
elif num1 == "miercoles":
    mensaje = "Miercoles tienes fisica"
elif num1 == "jueves":
    mensaje = "Jueves tienes quimica"
elif num1 == "viernes":
    mensaje = "Viernes tienes historia"
elif num1 == "sabado":
    mensaje = "Sabado tienes deportes"
elif num1 == "domingo":
    mensaje = "Domingo es tu dia de descanso Felicidades!!!"
else:
    mensaje = "No es un día valido"

print(mensaje)

File: Tema 3 -Sentencia condicional (if-switch)/ejercicios/ejercicio1 dia de la semana y que materia toca/archives/1.js
function sumarJS() {
  const num1 = (document.getElementById("num1").value);
  /*   let solucion; */
  switch (num1) {
    case "lunes":
      document.getElementById("resultadoJavascript").innerHTML = "Lunes tienes matematicas";
      break;
    case "martes":
      document.getElementById("resultadoJavascript").innerHTML = "Martes tienes lenguaje";
      break;
    case "miercoles":
      document.getElementById("resultadoJavascript").innerHTML = "Miércoles tienes fisica";
      break;
    case "jueves":
      document.getElementById("resultadoJavascript").innerHTML = "Jueves tienes quimica";
      break;
    case "viernes":
      document.getElementById("resultadoJavascript").innerHTML = "Viernes tienes musica";
      break;
    case "sabado":
      document.getElementById("resultadoJavascript").innerHTML = "Sábado tienes psicologia";
      break;
    case "domingo":
      document.getElementById("resultadoJavascript").innerHTML = "Domingo es tu dia de descanso Felicidades!!!";
      break;
    default:
      document.getElementById("resultadoJavascript").innerHTML = "No es un día valido";
  }

  /*   const resultadoDiv = document.getElementById("resultadoJavascript");
    resultadoDiv.innerHTML = `${solucion} `; */
}
